x-pack/legacy/plugins/canvas/public/plugin.tsx

```diff
@@ -43,9 +43,6 @@ export interface CanvasStartDeps {
     setRootController: Chrome['setRootController'];
```

Comment:
Can `setRootController` also be removed? Think we're just using the `application.register` now?